e2e/_suites/fleet/fleet.go

```diff
@@ -58,20 +56,17 @@ func (fts *FleetTestSuite) afterScenario() {
 	serviceName := common.ElasticAgentServiceName
 
 	if !fts.StandAlone {
-		agentInstaller := fts.getInstaller()
+		agentInstaller, _ := fts.deployer.Mount(serviceName, fts.InstallerType)
```

Comment:
I like the name of this method, as we will be mounting the agent installer into the service